packages/dds/test-dds-utils/assertTagging.config.mjs

```diff
@@ -0,0 +1,9 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * @type {import("@fluid-tools/build-cli").AssertTaggingConfig}
+ */
+export default { assertionFunctions: {} };
```

Comment:
Consider a comment explaining this disables assert tagging, since that's not obvious.

Reply:
done.